src/azure-cli/azure/cli/command_modules/monitor/operations/autoscale_settings.py

```diff
@@ -130,7 +145,11 @@ def autoscale_update(instance, count=None, min_count=None, max_count=None, tags=
         notification.email.send_to_subscription_administrator = email_administrator
     if email_coadministrators is not None:
         notification.email.send_to_subscription_co_administrators = email_coadministrators
-
+    predictive_policy = instance.predictive_autoscale_policy
+    if scale_mode is not None:
+        predictive_policy.scale_mode = scale_mode
+    if scale_look_ahead_time is not None:
+        predictive_policy.scale_look_ahead_time = scale_look_ahead_time
```

Comment:
When predictive_policy is None, it will fail in line 150 and 152

Reply:
done